.github/autoissuelabeler.yml

```diff
@@ -20,6 +20,9 @@
 - labels: [area/maven]
   title: "(?i).*maven.*"
   notify: [quarkusio/devtools]
+- labels: [area/jbang]
+  title: "(?i).*jbang.*"
+  notify: [quarkusio/devtools,maxandersen]
```

Comment:
Aren't you already in the `quarkusio/devtools` group?

Reply:
true - its probably redundant ;)

Reply:
but wanted to add in to see if it changes the notification setup i have. i'll remove it if the false positive gets too much.